wire-gradle-plugin/src/test/projects/since-until/src/main/proto/squareup/media/news.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto2";
+
+import "wire/extensions.proto";
+
+package squareup.media;
+
+option java_package = "com.squareup.media";
+
+message NewsFlash {
+  optional string title = 1;
+  optional string author = 2;
+  optional string radio = 3 [wire.until=1990];   // Omit this field on versions >= 1990.
+  optional string tv = 4 [wire.since=1950];      // Omit this field on versions < 1950.
+  optional string website = 5 [wire.since=2000]; // Omit this field on versions < 2000.
```

Comment:
is the wire prefix necessary?  `since` and `until` read nicer.

Reply:
Future proofing from conflicts.

Also, it looks like the spec required the parentheses for extensions https://developers.google.com/protocol-buffers/docs/proto

Reply:
Fixed!